### ISSUE TITLE
Syntax highlighting does not change the color of the full comment

### ISSUE THREAD

**Issue by jclsn** [daeyun/vim-matlab#33]
![image](https://user-images.githubusercontent.com/19779511/91890948-80e29480-ec90-11ea-8595-dc230240311e.png)

**Comment by MattiaSilvestri** [comment]
I have the same issue, did you manage to solve it?

**Comment by jclsn** [comment]
No, in fact I ditched all Matlab plugins for Vim, because they are buggy and not maintained anymore.
